src/video/emscripten/SDL_emscriptenevents.c

```diff
@@ -940,8 +975,8 @@ static void Emscripten_set_drag_event_callbacks(SDL_WindowData *data)
                         const file_reader = new FileReader();
                         file_reader.readAsArrayBuffer(file);
                         file_reader.onload = function(event) {
-                            const fs_dropdir = `/tmp/filedrop/${SDL3.drop_count}`;
-                            SDL3.drop_count += 1;
+                            const fs_dropdir = `/tmp/filedrop/${window_data.drop_count}`;
```

Comment:
You probably need to add the window ID here, so multiple windows don't clobber each other's path.